consensus/polybft/consensus_runtime.go

```diff
@@ -258,17 +258,17 @@ func (c *consensusRuntime) OnBlockInserted(fullBlock *types.FullBlock) {
 	postBlock := &PostBlockRequest{FullBlock: fullBlock, Epoch: epoch.Number, IsEpochEndingBlock: isEndOfEpoch}
 
 	// handle commitment and proofs creation
-	if err := c.stateSyncManager.PostBlock(postBlock); err != nil {
```

Comment:
Please revert all the code style changes like this. It gets tricky to review this alongside business logic.

Reply:
Done